bootloader/src/pyi_utils.c

```diff
@@ -183,10 +183,14 @@ pyi_setenv(const char *variable, const char *value)
 
 #ifdef _WIN32
     wchar_t * wvar, *wval;
+
     wvar = pyi_win32_utils_from_utf8(NULL, variable, 0);
     wval = pyi_win32_utils_from_utf8(NULL, value, 0);
 
-    rc = SetEnvironmentVariableW(wvar, wval);
+    // Not sure why, but SetEnvironmentVariableW() didn't work for me
+    //rc = SetEnvironmentVariableW(wvar, wval);
```

Comment:
Please remove this out-commented line. There is no use leaving out-commented code in the source. If necessary we can view the old code in the git repo.